source/_posts/README.md

```diff
@@ -0,0 +1,12 @@
+
+**Images**
+
+<!-- Add embedded images to your blog first by saving them to the posts folder within images (folder location: dev.waybetter.com/source/images/posts) -->
+
+<!-- Your image code should look like this:  -->
+
+<img class="post-images" src="/images/posts/your-photo.jpg" alt="Me" style="width:128px;height:128px;"> 
```

Comment:
does this folder exist or will the first person have to make it?  If it doesn't exist, how do we know that it's working
Also why are there inline styles in the README.md?  guessing @jasonrsavino would not approve.   So probably just find a good placeholder image and use that (instagram.com/poodlethebulldog is a good start. ;))